### ISSUE TITLE
Should forecasted values be restriced to the max capacity?

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/nowcasting_forecast#124]
Should we restrict our forecast values to max capacity of the GSP? 

This will stop crazy [bleeps](https://github.com/openclimatefix/nowcasting_forecast/issues/123) happening on the forecast, but it might cause things to fail silently? 

Perhaps max capacity of 0, we dont limit

What do you think @JackKelly and @jacobbieker ?

**Comment by JackKelly** [comment]
Sounds sensible to me! (although I must admit I haven't given it a huge amount of thought!)

**Comment by jacobbieker** [comment]
Yeah, limiting to max capacity makes sense to me. We just might want to add something to update the max capacity over time? Like every month or something?

**Comment by peterdudfield** [comment]
yea, the monthly update is on this issue https://github.com/openclimatefix/GSPConsumer/issues/43
